### PR TITLE
Fix missing roles in prometheus kubernetes example

### PR DESCRIPTION
**Pull request by zoidbergwill**
Needed with Prometheus 1.0

### CODE REVIEW

File: documentation/examples/prometheus-kubernetes.yml [hunk 1]
```diff
@@ -60,6 +61,7 @@ scrape_configs:
   - api_servers:
     - 'https://kubernetes.default.svc'
     in_cluster: true
+    role: endpoint
 
   relabel_configs:
   - source_labels: [__meta_kubernetes_role, __meta_kubernetes_service_annotation_prometheus_io_scrape]
```
Comment (jimmidyson) [inline documentation/examples/prometheus-kubernetes.yml]: You can also drop the `__meta_kubernetes_role` in each of the `source_labels`.

Comment (jimmidyson) [inline documentation/examples/prometheus-kubernetes.yml]: And the corresponding `true;` in the regexp.

Comment (anioool) [inline documentation/examples/prometheus-kubernetes.yml]: I see potential problem for the job kubernetes-cluster as there is a combination of two roles. If you set role to apiserver directly, you will not have entries for nodes targets.

Comment (jimmidyson) [inline documentation/examples/prometheus-kubernetes.yml]: Good point - this needs a another job with `role: node`.

Comment (zoidbergwill) [inline documentation/examples/prometheus-kubernetes.yml]: Awesome, thanks. Will do.

**Comment by jimmidyson** [comment]
:+1:

**Comment by matthiasr** [comment]
👍

**Pull request merged**